### DISCUSSION TITLE
Possibility to simple draw/mark something on screenshot

### DISCUSSION THREAD

**Discussion by idovecer** [elementary/screenshot-tool#236] [Ideas]
Will it be possible in the future to simple/easy draw or mark on a screenshot before saving to png?

Thank you.

**Comment by jeremypw** [comment]
Rather than expanding the scope of Screenshot and reinventing the wheel, maybe adding a "Open With" function would be better.  Then the user could install an app like Annotator to modify the image in whatever way they wish.

**Reply by idovecer** [reply]
I agree.

**Reply by danirabbit** [reply]
Following up here, I have a branch that pushes a simpler notication-based workflow that encourages opening in Photos: https://github.com/elementary/screenshot/pull/253
